server/neptune/workflows/internal/deploy/terraform.go

```diff
@@ -0,0 +1,17 @@
+package deploy
+
+import (
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/terraform"
+	"go.temporal.io/sdk/workflow"
+)
+
+// Export anything that callers need such as requests, signals, etc.
+type TerraformRequest = terraform.Request
+
+var TerraformTaskQueue = terraform.TaskQueue
+
+// To be able to execute a child workflow within the deploy workflow and avoid import cycles, terraform workflow
+// is defined within deploy package
+func Terraform(ctx workflow.Context, request TerraformRequest) error {
+	return terraform.Workflow(ctx, request)
+}
```

Comment:
Why are there import cycles? we shouldn't be referencing things in the deploy package from the terraform package? If we are let's move them, I would rather us keep a clean structure of one workflow per package.

Reply:
I think the import cycle was higher (deploy package shouldn't reference things defined in the internal package) like using an exported version of the terraform workflow request (`type TerraformRequest = terraform.Request`).

I can move it back to the original structure, but it just means the deploy package needs to be able to directly reference the terraform package. So really, we wouldn't need this file at all to act as the "external" representation of the workflow.